domain/src/main/java/com/example/domain/Order.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.domain
+
+
+class Order()
+{
+    private object Holder {
```

Comment:
For singleton pattern in Kotlin: https://kotlinlang.org/docs/tutorials/kotlin-for-py/objects-and-companion-objects.html